src/test/java/org/wso2/lsp4intellij/utils/UtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.lsp4intellij.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+  @Test
+  public void testConcatenateArrays() {
+    Assert.assertArrayEquals(new Object[]{new String[]{"a", "b"},
```

Comment:
You can do `new String[][] {{"a", "b"}, {"c", "d"}}` instead of using nested `String[]` in `Object[]`